pilot/pkg/model/sidecar.go

```diff
@@ -401,13 +401,9 @@ func (ilw *IstioEgressListenerWrapper) selectServices(services []*Service) []*Se
 					//   if service has multiple ports none of which match the listener port, check if there is
 					//   a virtualService with match Port
 					importedServices = append(importedServices, s)
-					hostFound = true
 					break
 				}
 			}
```

Comment:
Could you add few more comments - "in 1.1.5  .... happened, the behavior changed in 1.1.6 is ..., we no longer ignore ...."